subworkflows/local/samplesheet_to_channel/main.nf

```diff
@@ -0,0 +1,290 @@
+workflow  SAMPLESHEET_TO_CHANNEL{
+
+    take:
+    ch_from_samplesheet
+
+    main:
+    ch_from_samplesheet.dump(tag:"ch_from_samplesheet")
```

Comment:
We should really have some more dump operator everywhere.

BTW, thanks @RaqManzano for that subworkflows, the code reads nicer in the main workflow thanks to that